packages/components/spinner-loader/src/spinner-loader.ts

```diff
@@ -0,0 +1,49 @@
+import { buildProps } from '@puik/utils'
+import type { ExtractPropTypes, PropType } from 'vue'
+import type SpinnerLoader from './spinner-loader.vue'
+
+export enum spinnerSizes {
```

Comment:
Can you prefix it with `Puik` please ? `PuikspinnerSizes `
Same for other sizes